### PR TITLE
build: release 5.0.4 - undefined checks for MyInfo address fields

### PR DESCRIPTION
**Pull request by mantariksh**
Fixes issue where address components may be undefined.

### CODE REVIEW

File: src/app/modules/myinfo/myinfo.format.ts [hunk 1]
```diff
@@ -57,26 +57,32 @@ export const formatAddress = (addr: MyInfoAddress | undefined): string => {
...
+  }
 
-  buildingBlocks.push(postal.value)
+  if (postal?.value) {
+    buildingBlocks.push(postal.value)
+  }
 
   // Return string form with each block being separated by a space.
-  return buildingBlocks.filter((b) => b !== '').join(' ')
+  return buildingBlocks.join(' ')
```
Comment (liangyuanruo) [inline src/app/modules/myinfo/myinfo.format.ts:85]: why the change here?

**Review by liangyuanruo — APPROVED** [review]
lgtm. i see the releases made their way into the changelog again...

File: src/app/modules/myinfo/myinfo.format.ts (hunk 1 above)
Comment (mantariksh) [reply src/app/modules/myinfo/myinfo.format.ts:85]: if we check that all the values are truthy before pushing them into the array, then none of them can be empty strings
